lexer.go

```diff
@@ -360,9 +360,9 @@ func startWithSlash(s *Scanner) (tok int, pos Pos, lit string) {
 			break
 		}
 		s.r.inc()
-		// in '/*T!', try to consume the 5 to 6 digit version string.
-		commentVersion := s.scanVersionDigits(5, 6)
-		if commentVersion <= CommentCodeCurrentVersion {
+		// in '/*T!', try to match the pattern '/*T![feature1,feature2,...]'.
+		features := s.scanFeatureIDs()
+		if SpecialCommentsController.ContainsAll(features) {
 			s.inBangComment = true
 			return s.scan()
 		}
```

Comment:
What happens if the feature id is not included in the list?

Reply:
It will parse it as normal comments. For now, the behavior is:
`FeatureMap: {"xxx", "yyy"}`

```
/*T![xxx,yyy] part_of_sql */  # all the feature ids matched
/*T![xxx,zzz] part_of_sql */  # one feature id mismatched
/*T![xxx part_of_sql */       # no feature id is detected
/*T!part_of_sql */            # no feature id detected
```
are equivalant to 
```
part_of_sql

[xxx part_of_sql
part_of_sql
```
respectively.